src/main/java/org/webjars/FileSystemCache.java

```diff
@@ -21,11 +23,20 @@
     private Map<String, Cacheable> touched;
     private boolean dirty;
 
+    /**
+     * Create a file system cache.
+     *
+     * @param cache The file to load and store the cache to.
+     * @throws IOException If an error occurs.
+     */
     public FileSystemCache(File cache) throws IOException {
         this.cache = cache;
         reset();
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
you don't need the `@inheritDoc` because the docs will be inherited automatically if you don't specify your own. `@inheritDoc` is only for when you want to **include** the parent docs